cmd/virt-launcher/virt-launcher.go

```diff
@@ -348,9 +346,9 @@ func main() {
 	runWithNonRoot := pflag.Bool("run-as-nonroot", false, "Run virtqemud with the 'virt' user")
 	hookSidecars := pflag.Uint("hook-sidecars", 0, "Number of requested hook sidecars, virt-launcher will wait for all of them to become available")
 	ovmfPath := pflag.String("ovmf-path", "/usr/share/OVMF", "The directory that contains the EFI roms (like OVMF_CODE.fd)")
-	qemuAgentSysInterval := pflag.Duration("qemu-agent-sys-interval", 120*time.Second, "Interval between consecutive qemu agent calls for sys commands")
+	_ = pflag.Duration("qemu-agent-sys-interval", 120*time.Second, "[Deprecated] Interval between consecutive qemu agent calls for sys commands")
```

Comment:
Would it make sense to add note: this parameter is ignored? Since deprecated doesn't mean ignored.

Reply:
If it's not used anymore I'm voting to remove it. Why should we keep it around?

Reply:
Yes, if it is not used, then remove it completelly :+1:

Reply:
@codingben +1 for removing these flags if they aren't used anymore.

Reply:
Okay let's remove them. I thought that in case of backporting this will give user error if the flags were removed.

Reply:
> Also, I'm not sure we should change the polling intervals at all. With the current change we are reducing the interval for sys commands from 120 to 10 secs. How does this impact the load of the VM?

Yes you're right, I changed it back to 120s.